static/docs/commands-reference/repro.md

```diff
@@ -98,6 +98,13 @@ local cache and updates stage files with the new checksum information.
 
 - `--downstream` - rerun the commands present in the downstream of the pipeline.
 
+- `R, --recursive` - the `targets` value is expected to be a directory path.
```

Comment:
mind the comment in the issue. semantics has changed, and now it accepts files _and_ directories. Please, confirm that

Reply:
`-R` must be updated in other relevant commands, because it now accepts files as well. Again, check the issue and comments, please.

Reply:
now the -R can accept targets also right. SO should I add that it can accept targets(directory) and files also now right in other commands which I so far found is `dvc push and pull`

Reply:
-R was always accepting targets. The difference is that now you can pass a directory as well as a file, or mix of those. This is true for this command. This is now true for gc, push, pull (some other commands?)

Reply:
Seems like in [dvc gc](https://dvc.org/doc/commands-reference/gc) - The usage of -R is not documented

Reply:
my bad, gc does not have it indeed. Please, check other commands.

Reply:
Ok will check it out

Reply:
``R, --recursive` - the `targets` value is expected to be a directory path or
  a path corresponding to a specific file.`

@shcheklein are you ok with this sentence, if yes I will make change regarding it in DVC push, pull and bc command

Reply:
I would say something like (not literally) targets - directories, but also files. If target is directory - ... . For targets that are files it behaves the same way ...